demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -70,3 +73,52 @@ def test_is_valid_all_inputs_in_use(content_item, expected_result):
         if isinstance(expected_result, list)
         else result[0].message == expected_result
     )
+
+
+@pytest.mark.parametrize(
+    "content_item, expected_result",
+    [
+        (
+            create_playbook_object(),
+            [],
+        ),
+        (
+            create_playbook_object(
+                paths=["rolename"],
+                values=[[]],
+            ),
+            [],
+        ),
+        (
+            create_playbook_object(
+                paths=["rolename"],
+                values=[["Administrator"]],
+            ),
+            "The playbook 'Detonate File - JoeSecurity V2' can not have a rolename.",
+        ),
```

Comment:
keep in mind that content_items can be a list of items so I'd suggest splitting to two test cases: success and failure.